README.md

```diff
@@ -47,7 +47,7 @@ e-mail contact: security@arduino.cc
 Detailed instructions for installation in popular operating systems can be found at:
 
 -  [Linux](https://www.arduino.cc/en/Guide/Linux) (see also the [Arduino playground](https://playground.arduino.cc/Learning/Linux))
--  [macOS](https://www.arduino.cc/en/Guide/MacOSX)
+-  [macOS](https://www.arduino.cc/en/guide/MacOSX)
```

Comment:
```suggestion
-  [macOS](https://www.arduino.cc/en/Guide/macOS)
```
This is the modern page, you can find linked from the "Getting Started" home page:
https://www.arduino.cc/en/Guide#install-the-arduino-desktop-ide